### ISSUE TITLE
Wrong location for "new" gearbits

### ISSUE THREAD

**Issue by reconman** [anishlr/HyperLightDrifterLocator#1]
The gearbit with ID -1907892 is shown as a combat drop in the Eastern zone. It actually is a combat drop in the Northern zone with the video timestamp https://youtu.be/eqJS4c7n8Xs?t=3m35s

The current video link to -1907892 shows the location of -67920. The video link to -67920 shows the location of -917792. The video for -917792 shows the location of -907251.

You should double check all the timestamps from the final secrets video as most of them are wrong. I found the true location using http://codepen.io/smrq/pen/GZzMag

**Comment by anishlr** [comment]
Hmm interesting. Thanks for providing me an example. I'll look into it.

**Comment by reconman** [comment]
Non-combat gear bit locations seem to be fine as I was able to collect all missing gearbits in the North and West which were about 20 gear bits hidden in boxes and the one combat gear bit mentioned in my first comment.
